testing/integration-neo4j/src/test/java/com/neueda/jetbrains/plugin/graphdb/test/integration/neo4j/tests/jetbrains/StatementCollectorTest.java

```diff
@@ -123,7 +184,17 @@ public void testMultipleCorrectQueriesInDifferentLinesWithParameters() {
     }
 
     public void testParameterExtractionErrorOccurs() {
-        parametersService.registerParametersProvider(() -> WRONG_PARAMETERS);
+        parametersService.registerParametersProvider(new ParametersProvider() {
```

Comment:
Extract to test setup? It's just duplicated in every test.

Reply:
fixed in [d787900e](https://github.com/ex3m1024/jetbrains-plugin-graph-database-support/commit/d787900e73bc18d48e9b0ac186e839f178cb809b)